### DISCUSSION TITLE
How can we install Webstudio on Azure?

### DISCUSSION THREAD

**Discussion by pooriaarab** [webstudio-is/webstudio-community#173] [Q&A]
There was a link to deploy on netlify but not sure how we can deploy it to our azure instance.

Would appreciate any support on how to do that.

**Comment by kof** [comment]
We don't have content for azure, as we don't use it but here is:
- builder installation https://docs.webstudio.is/contributing/contributing-for-developers
- publishing of the project https://docs.webstudio.is/university/self-hosting-and-cli
